dashboard/app/models/applab.rb

```diff
@@ -37,6 +37,7 @@ class Applab < Blockly
     start_html
     encrypted_examples
     data_tables
+    data_kv
```

Comment:
data_kv --> data_properties

it's a little confusing, but within Applab we're calling these "Key/Value pairs" since that's how we expose them, but in the ruby and API layers we're calling them "properties".